_layouts/page.html

```diff
@@ -23,6 +23,7 @@ <h4 class="usa-accordion__heading desktop:display-none">
       {% include sidenav.html collection=page.subnav %}
       </div>
       {% endif %}
+      {% include expand-button.html %}
```

Comment:
Does/should this show up on mobile?

Reply:
It does and I think it should since a lot of users are on mobile devices and may want to use it cc: @nguyen-tina

Reply:
sorry, you're right - for some reason I was under the impression that the mobile menu was pre-expanded, but it's definitely not.